lib/views/status.erb

```diff
@@ -18,6 +18,28 @@
 </head>
 <body>
   <div class="container mt-3">
+    <div class="row">
+      <div class="col-md-12">
+        <% if !@notice.nil? %>
```

Comment:
Curious why `!@notice.nil?` rather than something like `@notice.present?`?

Reply:
@raghubetina My reasoning was that `present?` isn't a defined method since `web_git` doesn't load Rails. But now that I think about it... 🤔  the `active_support` gem is installed so I can `require` it and use `present?` instead. Thanks for pointing this out.